CMakeLists.txt

```diff
@@ -23,9 +23,25 @@ endif(MSVC)
 
 # gnu compiler or clang
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -mfpmath=sse -msse2 -std=c++14")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14")
 endif(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 
+if(SSE2_FOUND)
```

Comment:
Tohle si myslim, ze neni standardni CMake promenna. Alespon jsem ji nenasel na https://cmake.org/cmake/help/v3.9/manual/cmake-variables.7.html

Reply:
Trosku jsem to upravil